### PR TITLE
harmonize styling and content

### PR DESCRIPTION
**Pull request by bschmoker**
@MarkDavidson  cc:@gtback

(Similar to https://github.com/CybOXProject/cyboxproject.github.io/pull/11)

- Merged the /getstarted/intro page into a new /about page, since that's basically what it was
- moved para on 'taxii services' from /getstarted/intro to /docs/faq

cc:@taxiiproject

### CODE REVIEW

**Comment by gtback** [comment]
@bschmoker I only see the changes to the navbar, nothing about `/gettingstarted/intro` or `/about`.

Also my [comment](https://github.com/CybOXProject/cyboxproject.github.io/pull/11#discussion_r30613729) about CybOX applies here too.

**Comment by bschmoker** [comment]
@gtback - will push in a sec

File: getting-started/index.md [hunk 1]
```diff
@@ -21,7 +13,7 @@ tagline: User and Developer documentation for TAXII
     </div>
   </div>
 
-  <div class="col-md-6 getting-started-right">
+  <div class="col-md-6 ">
```
Comment (MarkDavidson) [inline getting-started/index.md]: When I render locally, the boxes aren't aligned:

image

Can this be fixed?

**Comment by MarkDavidson** [comment]
Just one comment. Everything else looks good.
-Mark

**Comment by bschmoker** [comment]
Ah, just saw the comment - Look cool to merge? @MarkDavidson

**Comment by gtback** [comment]
+1. If it were up to me (and it isn't) I would change a couple things:

* Change the word "impetus" to something else (not sure what)
* Move the "About" link to first in the navbar rather than last (same with STIX and CybOX). 

But I would be fine with this getting merged as is.

Also, should we add the "TAXII is being transitioned to OASIS" banner?

**Pull request merged**